rules.md

```diff
@@ -76,17 +54,16 @@ To ensure that the Incubator's processes and information remain fully open-sourc
 5. All people's work and rewards are public and verifiable
 6. All rules governing the Incubator and decisions on how to maintain those rules are public
 
-The only exception would be user credentials, i.e. their login passwords (which won't be needed when the Incubator App moves onto Dash Platform)
-
 ## 1.5 Users
 
-There are several levels to classification of Incubator App users as follows:
+There are several types of Incubator users, as follows:
 
-- Users - anyone who accesses the Incubator App or its underlying data
-- Members - any user who has signed up to create an account on the App
+- Users - anyone who accesses the Incubator or its underlying data
+- Members - any user who has signed up to the Incubator
 - [Contributors](#3-contributing) - Members who create output for the Incubator
-- [Admins](#4-admins) - Members with rights to create and manage Bounties, and earn commission from Task rewards
-- [Proposal Owner (PO)](#53-proposal-owner) - an Admin who is designated to raise Proposals to the Dash network on behalf of the Incubator 
+- [Admins](#4-admins) - Members with rights to create and manage Bounties
```

Comment:
Admins also create bounties? or only Strategists can create bounties?

Reply:
Good catch.  I should change this to:

[Admins](#4-admins) - Members with rights to manage Bounties accepted by Strategists.

Reply:
Resolved in https://github.com/dashincubator/dash-incubator-rules/pull/11/commits/c4c94a5e12f23b7efe0e79e6b4c694488486ce9b